engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/DataflowErrorsTest.scala

```diff
@@ -208,5 +208,25 @@ class DataflowErrorsTest extends InterpreterTest {
         "(Syntax_Error.Error 'Unexpected expression')"
       )
     }
+
+    "report correct stack trace for errors thrown from a suspended default argument" in {
+      val code =
+        """import Standard.Base.IO
+          |import Standard.Base.Error.Error
+          |
+          |type My_Error
+          |    Value
+          |
+          |fn x ~y=(Error.throw My_Error.Value) =
+          |    if x == 0 then 0 else y
+          |
+          |main =
+          |    IO.println (fn 1).get_stack_trace_text
+          |""".stripMargin
+      eval(code)
+      consumeOut should contain(
+        "        at <enso> <default::Test.fn::y>(Test:7:10-35)"
```

Comment:
The reported location checked by this test used to be:
```diff
-        at <enso> <default::Test.fn::Name.Literal(name = y, isMethod = false, location = Some(IdentifiedLocation[location=Location[start=89, end=90], uuid=None]), passData = MetadataStorage[], diagnostics = DiagnosticStorage(diagnostics = List()), id = null)>(Internal)
+        at <enso> <default::Test.fn::y>(Test:7:10-35)
```

1. the name was wrongly too long displaying IR internals
2. the location was `Internal` as it was not set correctly